scripts/eval/runners/judge_llm_runner.py

```
@@ -0,0 +1,75 @@
+        if content.endswith("```"):
+            content = content[:-3]
+
+        return json.loads(content.strip())
+
+    except json.JSONDecodeError as e:
+        logger.error(f"Failed to parse JSON from LLM Judge: {content}")
+        return {
+            "accuracy_score": 0,
+            "consistency_score": 0,
+            "citation_score": 0,
+            "reasoning": f"JSON Parsing Error: {str(e)}",
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)